apps/shabbat/shabbat.star

```diff
@@ -15,155 +15,352 @@ load("time.star", "time")
+        )
+
+        return render.Root(
+            render.Column(
+                children = [
+                    message,
+                    progress_bar,
+                ],
+            ),
+        )
+    else:
+        return render.Root(
+            message,
+        )
 
 def get_schema():
-    # There are different methods for deciding when it's sufficiently nightfall
     end_method_options = [
         schema.Option(
-            display = "8.5 degrees below horizon",
-            value = EIGHT_AND_A_HALF_DEGREES_BELOW_HORIZON,
+            display = "Degress below horizon",
+            value = "degrees_below_horizon",
         ),
         schema.Option(
-            display = "50 minutes after sunset",
-            value = FIFTY_MINUTES_AFTER_SUNSET,
+            display = "Halakhic Minutes",
+            value = "halakhic_min",
         ),
         schema.Option(
             display = "Rabbeinu Tam",
-            value = RABBEINU_TAM,
+            value = "rabbeinu_tam",
+        ),
+        schema.Option(
+            display = "50 minutes after sunset",
+            value = "fifty_minutes",
         ),
     ]
-
     return schema.Schema(
         version = "1",
         fields = [
             schema.Location(
                 id = "location",
                 name = "Location",
-                desc = "Location for which to display the times of Shabbat",
+                desc = "Location for which to display the times of shabbat",
                 icon = "locationDot",
             ),
             schema.Toggle(
                 id = "weekend_only",
                 name = "Weekend only?",
-                desc = "Enable to only display on Fridays and Saturdays.",
-                icon = "gear",
+                desc = "Enable to only display on Fridays and Saturdays",
+                icon = "calendar",
                 default = DEFAULT_WEEKEND_ONLY,
             ),
             schema.Toggle(
                 id = "use_24h",
-                name = "Use 24 hour clock",
+                name = "Use 24 hour clock?",
                 desc = "Enable to display times using the 24 hour clock",
                 icon = "clock",
                 default = DEFAULT_USE_24_HOUR_CLOCK,
             ),
+            schema.Toggle(
+                id = "show_progress_bar",
+                name = "Show progress bar?",
+                desc = "Enable to display progress bar on shabbat",
+                icon = "eye",
+                default = DEFAULT_SHOW_PROGRESS_BAR,
+            ),
             schema.Dropdown(
                 id = "end_method",
-                name = "End time",
-                desc = "How to calculate the end time of Shabbat",
-                icon = "hourglass",
-                default = end_method_options[0].value,
+                name = "Method",
+                desc = "Method to calculate the end of shabbat",
+                icon = "moon",
+                default = end_method_options[1].value,
```

Comment:
This app currently has:
```
app shabbat has 67 installations and is installed on 67 devices
```

If you update this default, it will cause anyone who installed this app with the default to get the new setting instead. Is that desired?

Reply:
That's a good point.  I don't want to get into too many halakhic arguments.  I'll allow for the old default.